docs/docs/using-pants/advanced-target-selection.mdx

```diff
@@ -185,3 +185,58 @@ For other goals, you can leverage shell piping to partition the input targets in
 ```bash
 pants list :: | awk 'NR % 5 == 0' | xargs pants package
 ```
+
+## Referring to a group of targets
+
+If setting tags on individual targets is not feasible, there are a few other options available to refer to multiple targets.
+
+If you have an operation that you perform often on a certain group of targets, you can use the
+[cli](https://www.pantsbuild.org/2.19/reference/subsystems/cli) subsystem options to create shortcuts. For instance,
```

Comment:
FYI, I think internal links like this should be written as relative links, e.g.

```suggestion
[cli](../../reference/subsystems/cli) subsystem options to create shortcuts. For instance,
```

and similarly for the others below, if they remain.

Reply:
Oh yes, that's great, thank you! I noticed that doing https://www.pantsbuild.org/reference/subsystems/cli (omitting the version) no longer works.